bin/audit-svg-colors.js

```diff
@@ -16,36 +16,69 @@ const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const PALETTE = require( '@automattic/color-studio' );
 const chroma = require( 'chroma-js' );
-const _ = require( 'lodash' );
+
+/**
+ * Native Sort - replacement of ._sortBy Lodash function
+ *
+ * @returns Sorted array.
+ */
+const sortBy = () => {
+	return ( objA, objB ) => {
+		if ( objA.to.name > objB.to.name ) {
+			return 1;
+		} else if ( objB.to.name > objA.to.name ) {
+			return -1;
+		}
+		return 0;
+	};
+};
 
 /**
  * Palette color subsets
  */
 
+/**
+ * pickBy function is a replacement for the Lodash ._pickby
+ *
+ * @param {*} obj a palette of colors
+ * @param {*} fn This function returns truthy or falsy value to filter the colors
+ * @returns objext with filtered color values
+ * Used Object.keys() and Array.prototype.filter()to remove the keys for which fn returns a falsy value.
```

Comment:
Feel free to remove this line, I believe it should be pretty straightforward what it does by reading the code.